Resources/public/js/alloyeditor/buttons/embed.js

```diff
@@ -75,9 +75,14 @@ YUI.add('ez-alloyeditor-button-embed', function (Y) {
          */
         _addEmbed: function (e) {
             var contentInfo = e.selection.contentInfo,
+                scrollX = window.pageXOffset || document.documentElement.scrollLeft,
```

Comment:
As far as I know `document.documentElement.scrollLeft` and `document.documentElement.scrollTop` is for IE 8 and lower. We dropped support a long time ago for IE, so I think you can omit this.